observation_portal/observations/models.py

```diff
@@ -35,6 +35,9 @@ def observation_as_dict(instance, no_request=False):
 def get_expanded_configurations(observation, configurations):
     ''' Gets set of expanded configurations with configuration details filled in for a given observation
     '''
+    # If this is a REAL_TIME observation, just return its configuration_status since it doesn't have a configuration
+    if observation.request.request_group.observation_type == RequestGroup.REAL_TIME:
+        return [{'configuration_status': observation.configuration_statuses.first().id}]
```

Comment:
Is there something that ensures there's at least 1 configuration_statuses for a real time observation type? Otherwise, this might return an exception about accessing `.id` on `None`

Reply:
That's a good point - there will be from this PR (it is always created in the serializer now), but past REAL_TIME observations don't have them yet. I'll protect against that.